### PR TITLE
SLZ Fix

### PR DESCRIPTION
**Pull request by mattokc35**
Changed some helper functions which were causing landing zone to break and updated unit tests/test coverage accordingly

### CODE REVIEW

**Comment by mattokc35** [comment]
@jvallexm

File: unit-tests/helpers.test.js [hunk 1]
```diff
@@ -476,6 +476,23 @@ describe("helpers", () => {
...
+      let actualData = helpers.deepObjectIgnoreNullValues(undefined);
+      let expectedData = {};
+      assert.deepEqual(actualData, expectedData, "it should return object");
+    });
+    it("should not recurr if an object child value is null", () => {
+      let actualData = helpers.deepObjectIgnoreNullValues({
+        frog: { identifier: null },
+      });
+      let expectedData = { identifier: null };
+      //assert.deepEqual(actualData, expectedData, "it should return object");
```
Comment (jvallexm) [inline unit-tests/helpers.test.js]: reimplement assertion

File: unit-tests/tfx-cli.test.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ describe("cli", () => {
         actualData = data;
       };
       tfWithLogs.tfxcli();
-      assert.deepEqual(actualData, `tfx cli version 1.1.0`, "it should return correct data");
+      assert.deepEqual(actualData, `tfx cli version 1.2.0`, "it should return correct data");
```
Comment (jvallexm) [inline unit-tests/tfx-cli.test.js]: should be `1.1.1`

File: lib/extract.js [hunk 3]
```diff
@@ -205,9 +211,14 @@ const extract = {
       ? outputData
       : "\n" +
           YAML.stringify(outputData)
-            .replace(quotesAroundStringsAndEscapedSlashes, "") // remove quotes around string type variables and also escaped slashes
-            .replace(escapedQuotesRegExp, '"') // replace escaped quotes with single quotes
-            .replace(/\\\\/g, ""); //replace leftover slashes
+            .replace(/"-"/g, "@@@")
```
Comment (jvallexm) [inline lib/extract.js]: use `RegexButWithWords` here, ensure ut all passes

**Review by jvallexm — CHANGES_REQUESTED** [review]

**Pull request merged**